testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/util/IcebergTestUtils.java

```diff
@@ -18,14 +18,23 @@
 
 import static com.google.common.base.Verify.verify;
 import static io.trino.tests.product.utils.QueryExecutors.onTrino;
+import static java.lang.String.format;
 
 public final class IcebergTestUtils
 {
+    private static final String NAMENODE_URI = "hdfs://hadoop-master:9000";
+
     private IcebergTestUtils() {}
 
     public static String getTableLocation(String tableName)
     {
-        Pattern locationPattern = Pattern.compile(".*location = 'hdfs://hadoop-master:9000(.*?)'.*", Pattern.DOTALL);
+        return getTableLocation(tableName, false);
+    }
+
+    public static String getTableLocation(String tableName, boolean withNamenodeURI)
+    {
+        String regex = format(".*location = '%s(.*?)'.*", withNamenodeURI ? "" : NAMENODE_URI);
```

Comment:
It looks like "with namenode uri" parameter cause the namenode uri to be stripped, not retained?

Anyway, this is an unnnecessary complexity of this method.
It should retrieve the table location. The table location may start with `hdfs:///` and it's not this method's problem to deal with it.

Reply:
> It looks like "with namenode uri" parameter cause the namenode uri to be stripped, not retained?

It's other way around ("withNamenodeURI" does not strip namenode uri) . 

Anyway, I will try to simplify and remove the complexity from this method.

Reply:
Created a new method `stripNamenodeURI` to strip the namenode uri wherever needed.